src/ToastContainers.tsx

```diff
@@ -14,7 +14,7 @@ const ToastContainers: FC = (): JSX.Element => {
         pauseOnFocusLoss
         draggable
         pauseOnHover
-        theme="light"
+        theme="dark"
```

Comment:
why did you change to dark theme? was that show to the PO? 🤔

Reply:
I thought it would work better with the other popups that are not toast popups. We probably should make them all similiar because some use mui alerts still i think

Reply:
Also there are some conflicting files